boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -62,6 +62,46 @@ The toolchain is available for Linux, Windows, and Mac hosts and
 instructions to obtain and set them up are available in the ESP-IDF
 repository, using the toolchain and SDK links above.
 
+Docker container based builds
+=============================
+
```

Comment:
You've left out explaining how to install the docker tools needed to run your pre-made image, e.g., the ``docker`` command.

Reply:
yeah - I guess it might be worth adding a link to the Docker install instructions - but on the other hand - perhaps its kinda like adding "install linux" instructions to the other cases.